src/definitions/internet.ts

```diff
@@ -1,5 +1,17 @@
 import { allOf } from './utils';
 
+export type EmojiGroups =
```

Comment:
```suggestion
export type EmojiGroup =
```

Personally I would call it a `type` and not a `group`, but that's only a side note...

Reply:
After a bit of thinking it really feels more like a `type`: `food`, `smiley`, `body`... and in the definitions, that just grouped by type. But what you want from the user in the method definitions is to _limit emoji to a certain type_...

Reply:
From the unicode source, this is call a `group`. Can you confirm that you want me to rename it to `type`?
cf https://unicode.org/Public/emoji/14.0/emoji-test.txt

![image](https://user-images.githubusercontent.com/4563971/162574972-680a2591-f267-4258-ac90-df598df19746.png)

Reply:
In the source file its called a group, which indeed collects emojis by type :) Its kind of like you would group cars by size: sedan, suv, etc. Collection of all SUV would be a group, but the attribute they are grouped by would be `size`. Does that make sense?

Reply:
And here they call them _category_ https://home.unicode.org/emoji/emoji-frequency/

Yeah, I am pretty sure it should be called differently then `group` as its too general word. You can group emojis in many different ways...

Reply:
@pkuczynski IMO since we use that source we should also stick to their names.
group and category are just synonmys. You could group/categorize them however you want, but we use the "official" groups.

If you have a better name for it, then we can decide on that then.

Reply:
Aaaah, I renamed it to type >< ... https://github.com/faker-js/faker/pull/504/commits/24727edfbe7c4f7863e77f91ccda147a9eba627f

Should I change it back?

Reply:
My better name is `Type` @ST-DDT :)

Reply:
> Should I change it back?

Nah, I think you can keep it, unless @ST-DDT disagrees?